x-pack/plugin/esql/src/main/java/generated/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvSliceBooleanEvaluator.java

```diff
@@ -0,0 +1,124 @@
+// or more contributor license agreements. Licensed under the Elastic License
+// 2.0; you may not use this file except in compliance with the Elastic License
+// 2.0.
+package org.elasticsearch.xpack.esql.expression.function.scalar.multivalue;
+
+import java.lang.Override;
+import java.lang.String;
+import org.elasticsearch.compute.data.Block;
+import org.elasticsearch.compute.data.BooleanBlock;
+import org.elasticsearch.compute.data.IntBlock;
+import org.elasticsearch.compute.data.Page;
+import org.elasticsearch.compute.operator.DriverContext;
+import org.elasticsearch.compute.operator.EvalOperator;
+import org.elasticsearch.core.Releasables;
+import org.elasticsearch.xpack.esql.expression.function.Warnings;
+import org.elasticsearch.xpack.ql.tree.Source;
+
+/**
+ * {@link EvalOperator.ExpressionEvaluator} implementation for {@link MvSlice}.
+ * This class is generated. Do not edit it.
+ */
+public final class MvSliceBooleanEvaluator implements EvalOperator.ExpressionEvaluator {
+  private final Warnings warnings;
+
+  private final EvalOperator.ExpressionEvaluator posistion;
+
+  private final EvalOperator.ExpressionEvaluator field;
+
+  private final EvalOperator.ExpressionEvaluator start;
+
+  private final EvalOperator.ExpressionEvaluator end;
+
+  private final DriverContext driverContext;
+
+  public MvSliceBooleanEvaluator(Source source, EvalOperator.ExpressionEvaluator posistion,
+      EvalOperator.ExpressionEvaluator field, EvalOperator.ExpressionEvaluator start,
+      EvalOperator.ExpressionEvaluator end, DriverContext driverContext) {
+    this.warnings = new Warnings(source);
+    this.posistion = posistion;
+    this.field = field;
+    this.start = start;
+    this.end = end;
+    this.driverContext = driverContext;
+  }
+
+  @Override
+  public Block eval(Page page) {
+    try (BooleanBlock fieldBlock = (BooleanBlock) field.eval(page)) {
+      try (IntBlock startBlock = (IntBlock) start.eval(page)) {
+        try (IntBlock endBlock = (IntBlock) end.eval(page)) {
+          return evalMv(page.getPositionCount(), fieldBlock, startBlock, endBlock);
+        }
+      }
+    }
+  }
+
+  public BooleanBlock evalMv(int positionCount, BooleanBlock fieldBlock, IntBlock startBlock,
+      IntBlock endBlock) {
+    try(BooleanBlock.Builder result = driverContext.blockFactory().newBooleanBlockBuilder(positionCount)) {
+      position: for (int p = 0; p < positionCount; p++) {
+        if (fieldBlock.isNull(p)) {
+          result.appendNull();
+          continue position;
+        }
+        if (startBlock.isNull(p)) {
+          result.appendNull();
+          continue position;
+        }
+        if (endBlock.isNull(p)) {
+          result.appendNull();
+          continue position;
+        }
+        result.beginPositionEntry();;
+        MvSlice.process(result, p, fieldBlock, startBlock.getInt(startBlock.getFirstValueIndex(p)), endBlock.getInt(endBlock.getFirstValueIndex(p)));
```

Comment:
I think we should return `null` if the second or third parameter are multivalued here. I think that's *likely* true for all of the mv_ functions